libs/client/ClientSession/GarnetClientSessionMigrationExtensions.cs

```diff
@@ -239,6 +240,10 @@ public Task<string> MigrateData(string sourceNodeId, Memory<byte> replaceOption,
         /// </summary>
         public bool InitMigrateCommand => curr == null;
 
+        int ExtraSpace =>
```

Comment:
Any reason why this is a property and not a readonly field? Just means that it would recalculate at every read...

Reply:
did not want to waste space in the GarnetClient, as the calculation is trivial addition.